### PR TITLE
Smoothened takeoff and landing gains for mttr-4.0

### PR DESCRIPTION
**Pull request by Aksellence**
Porting the changes mentioned here: https://github.com/matternet/ardupilot/commit/c3360eb3f6e320f593e24c0f9d0eec4d4aed2ce5 & https://github.com/matternet/ardupilot/commit/17438adeb0cd8d02d7da0d4e18e2651d303ce817 to mttr-rebase-4.0.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm [hunk 1]
```diff
@@ -1,263 +1,263 @@
...
+BATT_LOW_MAH	0
+LOIT_ANG_MAX	20
+PSC_ACCZ_D	0
+PSC_ACCZ_FF	0
+PSC_ACCZ_FLTE	15
+PSC_ACCZ_I	0.5
+PSC_ACCZ_IMAX	800
+PSC_ACCZ_P	0.2
+PSC_ANGLE_MAX	0
+PSC_POSXY_P	1.35
```
Comment (zalice) [inline libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm]: @AkshayMatternet Could you please set the defaults to:
`PSC_POSXY_P 0.75
PSC_VELXY_I 0.75
PSC_VELXY_P 1.5
`

File: libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm [hunk 2]
```diff
@@ -1,263 +1,263 @@
...
+ANGLE_MAX	2000
+ARMING_CHECK	16318
+ATC_ACCEL_P_MAX	30000
+ATC_ACCEL_R_MAX	36000
+ATC_ACCEL_Y_MAX	10000
+ATC_ANG_LIM_TC	3
+ATC_ANG_PIT_P	6
+ATC_ANG_RLL_P	6
+ATC_ANG_YAW_P	5
+ATC_RAT_PIT_D	0.01
```
Comment (zalice) [inline libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm]: Could you please also set these defaults?
`ATC_ACCEL_P_MAX    30000
ATC_RAT_PIT_D  0.008
ATC_RAT_PIT_P  0.11
ATC_RAT_RLL_D  0.007
ATC_RAT_RLL_P  0.1
ATC_RAT_YAW_I  0.0888
ATC_RAT_YAW_P  0.8
`

File: libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm (hunk 1 above)
Comment (Aksellence) [reply libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm]: Yes I can, I was planning to add those params as a separate PR, to make it easy for comparison, between 3.6.7 and 4.0

Comment (zalice) [reply libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm]: Thanks. That's fine

**Review by zalice — APPROVED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm (hunk 1 above)
Comment (Aksellence) [reply libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm]: Added the changes, and are up for review. Thanks @zalice

File: libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm (hunk 2 above)
Comment (Aksellence) [reply libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm]: Added the changes and are up for review.

**Pull request merged**